test/new/db/formats/coff

```diff
@@ -6,34 +6,34 @@ iS
 EOF
 EXPECT=<<EOF
 [Imports]
-nth vaddr      bind type lib name
----------------------------------
-0   0x00000000 NONE FUNC     __TIFFmalloc
-1   0x00000000 NONE FUNC     __TIFFrealloc
-2   0x00000000 NONE FUNC     __TIFFmemset
-3   0x00000000 NONE FUNC     __TIFFmemcpy
-4   0x00000000 NONE FUNC     __TIFFfree
-5   0x00000000 NONE FUNC     _TIFFFindField
-6   0x00000000 NONE FUNC     _TIFFFieldWithTag
-7   0x00000000 NONE FUNC     _TIFFReadDirectory
-8   0x00000000 NONE FUNC     _TIFFErrorExt
-9   0x00000000 NONE FUNC     _TIFFWarningExt
-10  0x00000000 NONE FUNC     _TIFFSwabShort
-11  0x00000000 NONE FUNC     _TIFFSwabLong
-12  0x00000000 NONE FUNC     _TIFFSwabLong8
-13  0x00000000 NONE FUNC     __TIFFGetFields
-14  0x00000000 NONE FUNC     __TIFFGetExifFields
-15  0x00000000 NONE FUNC     __TIFFSetupFields
-16  0x00000000 NONE FUNC     __TIFFFillStriles
-17  0x00000000 NONE FUNC     __TIFFNoPostDecode
-18  0x00000000 NONE FUNC     sym.imp.BitData
-19  0x00000000 NONE FUNC     sym.imp.BitData
-20  0x00000000 NONE FUNC     sym.imp.BitData
-21  0x00000000 NONE FUNC     sym.imp.BitData
-22  0x00000000 NONE FUNC     _TIFFSetCompressionScheme
-23  0x00000000 NONE FUNC     __TIFFDataSize
-24  0x00000000 NONE FUNC     __TIFFCheckMalloc
-25  0x00000000 NONE UNK      __fltused
+nth vaddr bind type lib name
+----------------------------
+0   -1    NONE FUNC     __TIFFmalloc
```

Comment:
Is it ok?

Reply:
Hows that cant be resolved?

Reply:
What can't be resolved?

Reply:
> The address of the imports so the disassembly can use it to show meaningful calls. Imma bit out of coff and maybe those imports are loaded as relocs by the runtime linker. But in case the binary have a place to hold the trampolines to jump to the import like the elf plt this should be resolved there

In case of `obj` files imports are just names for relocs but binary doesn't have trampolines. Linker (like `ld`; not a runtime linker) modifies call/jump instructions directly inside of place of use.